### PR TITLE
machine/vrender0.cpp: Fix clock input match as XIN

### PR DESCRIPTION
**Pull request by cam900**

### CODE REVIEW

File: src/devices/machine/vrender0.cpp [hunk 1]
```diff
@@ -131,14 +131,14 @@ void vrender0soc_device::device_add_mconfig(machine_config &config)
 	m_screen->set_palette(m_palette);
 
 	// runs at double speed wrt of the CPU clock
-	VIDEO_VRENDER0(config, m_vr0vid, DERIVED_CLOCK(2, 1));
+	VIDEO_VRENDER0(config, m_vr0vid, DERIVED_CLOCK(6, 1));
```
Comment (angelosa) [inline src/devices/machine/vrender0.cpp:134]: Source?

**Review by angelosa — CHANGES_REQUESTED** [review]

File: src/devices/machine/vrender0.cpp (hunk 1 above)
Comment (cam900) [reply src/devices/machine/vrender0.cpp:134]: Mostly at datasheet.
CPU clock is XIN *3, Video clock is CPU clock * 2. (default, Internal PLL frequency?)

Comment (angelosa) [reply src/devices/machine/vrender0.cpp:134]: Other than the comments to update, I'm not eager to have this _for current release cycle_ for mostly the same reasons I waited to apply mine. The main question is: how it fares with donghaer split screen, knowing it's timing sensitive?

Comment (cam900) [reply src/devices/machine/vrender0.cpp:134]: I think bus clock (and CPU clock?) is configured by PLL, but PLL isn't emulated.

**Pull request closed without merging**